src/controllers/product.controller.js

```diff
@@ -0,0 +1,10 @@
+const productService = require("../services/product.service");
+const { catchAsync } = require("../utils/error");
+
+const getNewProducts = catchAsync(async (req, res) => {
+  const orderBy = req.query.orderBy;
+  const limitNum = req.query.limitNum;
```

Comment:
```suggestion
  const { orderBy, limitNum } = req.query
```
코드는 줄일 수 있다면 한 줄이라도 줄이는것이 좋습니다!